pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1856,7 +1856,14 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
           throw new Error('No container engine is set');
         }
         if (kubernetesVersion) {
-          await this.kubeBackend.install(config, kubernetesVersion, this.#adminAccess);
+          if (!config.kubernetes.version) {
+            const versionSetting = { kubernetes: { version: kubernetesVersion.version } };
+
+            await this.kubeBackend.install(merge(config, versionSetting), kubernetesVersion, this.#adminAccess);
```

Comment:
`merge` modifies its first argument. We probably want:
```suggestion
            await this.kubeBackend.install(merge({}, config, versionSetting), kubernetesVersion, this.#adminAccess);
```

Reply:
That's what happens when I take a week off, I forget lodash's little gotchas.